src/modules/addTodoTask.js

```diff
@@ -76,6 +45,15 @@ const renderTask = () => {
     const moreBtn = document.querySelectorAll('.toggle');
     const deleteIcon = document.querySelectorAll('.hideDelete');
     const editIcon = document.querySelectorAll('.hideEdit');
+    const completeTasks = document.querySelectorAll('.completeTask');
+
+    completeTasks.forEach((task) => {
+      // Attach event listeners to all checkboxes
+      task.addEventListener('change', () => {
+        // eslint-disable-next-line no-use-before-define
+        completeTask(task.id, task);
```

Comment:
- [ ] Kindly don't disable linters because linters prevent you from making those errors in your code which can break your whole app 😨 Kindly remove all the eslint disable lines and fix your linters manually 👍